front/src/api/today-register-list.api.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from "axios";
+import {baseRoute, routesLinks} from "core";
+
+export interface RegisterEntityApi {
+    cardNumber: string;
+    date: string;
+    time: string;
+    firstName: string;
+    lastName: string;
+    idNumber: string,
+    company: string;
+    host: string;
+    signature: string;
+    picture: string;
+}
+
+const registerCollection = `${baseRoute}${routesLinks.registerCollection}`;
+const today = new Date().toJSON().slice(0,10);
```

Comment:
It would be a good idea to separate this into a util file, something like getTodayDate

By following this approach we follow the same level of abstraction principle